test/testPriceRouter/SequencerPriceRouter.t.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.21;
+
+import { SequencerPriceRouter, PriceRouter, Registry, ERC20, IChainlinkAggregator } from "src/modules/price-router/permutations/SequencerPriceRouter.sol";
+import { Math } from "src/utils/Math.sol";
+
+import { Test, stdStorage, StdStorage, stdError, console } from "@forge-std/Test.sol";
+
+// TODO refactor to a multichain test once deploy is merged into branch.
+contract SequencerPriceRouterTest is Test {
+    using Math for uint256;
+    using stdStorage for StdStorage;
+
+    SequencerPriceRouter public sequencerPriceRouter;
+
+    address arbitrumSequencerUptimeFeed = 0xFdB631F5EE196F0ed6FAa767959853A9F217697D;
+    ERC20 public WETH = ERC20(0x82aF49447D8a07e3bd95BD0d56f35241523fBab1);
+    ERC20 public USDC = ERC20(0xaf88d065e77c8cC2239327C5EDb3A432268e5831);
+    address public USDC_USD_FEED = 0x50834F3163758fcC1Df9973b6e91f0F0F0434aD3;
+    Registry public registry;
+
+    function setUp() external {
+        // Setup forked environment.
+        string memory rpcKey = "ARBITRUM_RPC_URL";
+        uint256 blockNumber = 161730035;
+
+        uint256 forkId = vm.createFork(vm.envString(rpcKey), blockNumber);
+        vm.selectFork(forkId);
+
+        sequencerPriceRouter = new SequencerPriceRouter(address(this), 3_600, address(this), registry, WETH);
```

Comment:
For the `_gracePeriod` constructor arg, what would be a fair value to use? This seems to be specific to the protocol and design situation being considered.

Reply:
That makes sense. We also discussed the main question I had within the PR review comment about having the gracePeriod in lock-step with external protocols that L2 cellars interact with. We decided that some stagger would be okay for protocol integrations. We can touch on this more with Chainlink as they get back to us in the new year too.

Reply:
I'll leave this unresolved so we can make sure to discuss it with Macro cause I'm curious about their thoughts on it too.